modules/ocf_puppet/files/update-prod

```diff
@@ -17,7 +17,7 @@ else
 fi
 
 # update from remote
-git fetch origin
+git fetch origin master
 git reset --hard origin/master
 
 # Install third-party modules
```

Comment:
Don't you need a `make .resource_types` in here too now that it's moved to a different target?

Reply:
fuck